tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -370,6 +370,95 @@ TEST(RNTuple, StdSet)
    EXPECT_EQ(pairSet, *mySet2);
 }
 
+TEST(RNTuple, StdMap)
+{
+   auto field = RField<std::map<char, int64_t>>("mapField");
+   EXPECT_STREQ("std::map<char,std::int64_t>", field.GetType().c_str());
+   auto otherField = RFieldBase::Create("test", "std::map<char, int64_t>").Unwrap();
+   EXPECT_STREQ(field.GetType().c_str(), otherField->GetType().c_str());
+   EXPECT_EQ((sizeof(std::map<char, int64_t>)), field.GetValueSize());
+   EXPECT_EQ((sizeof(std::map<char, int64_t>)), otherField->GetValueSize());
+   EXPECT_EQ((alignof(std::map<char, int64_t>)), field.GetAlignment());
+   // For type-erased map fields, we use `alignof(std::map<std::max_align_t, std::max_align_t>)` to map the alignment,
+   // so the actual alignment may be smaller.
```

Comment:
I think that if this ever happens, it can become a problem.  I guess that in practice the alignment of inner items does not matter. Let's perhaps add an equality test with `map<char, char>` so that if we see an alignment mismatch, we notice it early in the unit tests.